docs/data/material/migration/migrating-from-deprecated-apis/migrating-from-deprecated-apis.md

```diff
@@ -120,6 +121,98 @@ The Alert's `componentsProps` was deprecated in favor of `slotProps`:
  />
 ```
 
+### Composed CSS classes
+
+The CSS classes that composed the `severity` (or `variant`) and `color` prop values were removed.
+
+Here's how to migrate:
+
+```diff
+-.MuiAlert-standardSuccess
++.MuiAlert-standard.MuiAlert-colorSuccess
```

Comment:
I've had this thought in the past as well. I will add it for discussion at the next Material UI meeting.

What's your take?

Reply:
> What's your take?

I believe this is acceptable because developers are likely accustomed to the classname without the "variant" string in the composed classes. Moreover, "variant" serves as the primary distinguishing prop among different types of a component. Thus, perhaps initially it wasn't prefixed with the "variant" string.

Reply:
That's a good point 👍🏼 

I've created an RFC to track the discussion: https://github.com/mui/material-ui/issues/41277

If you want to leave your opinion there it will be very valuable 😊